schedule/yast/maintenance/create_hdd_transactional_server_dev.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: create_hdd_transactional_server
+description: >
+    Installation of a Transactional Server which uses a read-only
+    root filesystem to provide atomic, automatic updates of a
+    system without interfering with the running system.
+vars:
+    DEPENDENCY_RESOLVER_FLAG: 1
+    DESKTOP: textmode
+    PATTERNS: base,minimal
+    HDDSIZEGB: 40
+    DUD_ADDONS: sdk
+    YUI_REST_API: 1
+schedule:
+    - installation/bootloader_start
+    - installation/setup_libyui
+    - installation/product_selection/install_SLES
+    - installation/licensing/accept_license
+    - installation/registration/register_via_scc
+    - installation/module_registration/register_module_desktop
+    - installation/add_on_product_installation/add_additional_products
+    - installation/add_on_product/skip_install_addons
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
```

Comment:
should use accept_selected_role_transactional_server? Since the name of test case if for transactional ?

Reply:
- installation/system_role/accept_selected_role_transactional_server